markdown/the_photomode_wishlist.md

```diff
@@ -16,7 +16,7 @@ These are the features that are a must for any photomode. Some may seem obvious,
 The following aspects regarding camera movement and orientation are crucial:
 
 - Maximum range as big as possible
-  : Maybe your game or engine is not prepared to deal with the camera being far away from the player, but try to push this value as much as you can. Small range limits are one of the things that get modded first on PC, so players will certainly appreciate it. Some game designers may be worried about players moving the camera around to find secrets or look at what comes next in the level, but this is a decision that comes from the player and in a photomode context the advantages outweigh the downsides.
+  : Maybe your game or engine is not prepared to deal with the camera being far away from the player, but try to push this value as much as you can. Small range limits are one of the things that get modded first on PC, so players will certainly appreciate it. Some studios may be worried about players moving the camera around to find secrets or look at what comes next in the level, but this is a decision that comes from the player and in a photomode context the advantages outweigh the downsides.
```

Comment:
Might just be my ignorance of the gaming industry, but would the studio execs really care about players using photomodes to "peek ahead"?

I guess their angle might be more about seeing camera clipping or unrendered areas or something and thinking that reflects poorly on the product/brand though... 🤷

Reply:
Sorry, I should have clarified why I changed this back. Yeah, I believe you are right in that game/level designers might care more about this than execs, and I am sure execs def care about clipping and taking the camera where we shouldn't. Thought that just writing studios might include all of the potential people that *might* have a problem with this, but I don't have a strong opinion about it, so feel free to ignore it.

Reply:
I'd go for studios, as 'team' isn't really a term used when a game is released.

Reply:
I'm sure I've heard "team" used before — "The team behind such-and-such" or "The team at whatever studio" — but I don't mean to belabor the point too much. "Studio" is fine.


Or maybe "stakeholders" 😉